### DISCUSSION TITLE
HTTP2 support

### DISCUSSION THREAD

**Discussion by mduller** [iterate-ch/cyberduck#14917] [Q&A] [answered]
Hi,

I have been searching for any mention of HTTP2 in conjunction with Mountain Duck but I could not find any information.
Is there a configuration, manual profile tweak, etc. that can be used to enable the use of HTTP2 when connecting to WebDAV (Nextcloud)?

Thanks!

**Comment by ylangisc** [comment, marked as answer]
There is currently no support for HTTP/2 in Cyberduck nor are there concrete plans to do so.

Sidenote: The library we use currently does not provide proxy support for HTTP/2 connections. [^1] 

[^1]: [Please note that presently HTTP/2 clients do not support request execution via an HTTP proxy.](https://hc.apache.org/httpcomponents-client-5.2.x/migration-guide/migration-to-async-http2.html)

**Reply by mduller** [reply]
Hi Yves,
Thanks for the quick response. I wasn't aware that proxy support for HTTP/2 is missing - makes sense to not activate partial HTTP/2 support.
Michael
